pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -496,6 +527,25 @@ private static void start() {
         LOG.info("ddp startup complete");
     }
 
+    private static void setupRateLimits(int maxRequestsPerSecond, int burst) {
```

Comment:
Consider moving the setup/wrapping of the DoSFilter to a separate class that can be `import`ed into DataDonationPlatform as just another filter that implements `spark.Filter` . DataDonationPlatform is kind of largish already.